### ISSUE TITLE
Dumbbell lowMarker is hidden when accessibility module is loaded and normal marker is disabled

### ISSUE THREAD

**Issue by kamil-musialowski** [highcharts/highcharts#25279]
#### Expected behaviour
`lowMarker` should be visible when accessibility module is loaded and normal `marker` is disabled

#### Actual behaviour
When accessibility module is loaded, lowMarker gets hidden

#### Live demo with steps to reproduce
Remove a11y module to see rendered marker
https://jsfiddle.net/BlackLabel/5zjw436e/

#### Workaround
Instead of setting `enabled` property, set`radius`.
https://jsfiddle.net/BlackLabel/aumj1vfe/

#### Product version
Core, Dumbbell, A11y - 13.0.1 (latest)

**Comment by kamil-musialowski** [comment]
Probably broken over here: https://github.com/highcharts/highcharts/blob/48b5b3e99a0fa9aae9a357165a5a916f504afeca/ts/Accessibility/Components/SeriesComponent/ForcedMarkers.ts#L249-L253
